docs/ledger-nano-s.md

```diff
@@ -35,14 +35,20 @@ These are some basic examples of commands you can use with your Ledger. You may
 ![](https://miro.medium.com/max/1536/1*Xfi5_ScAiFn6rr9YBjgFFw.jpeg)
 _Ref: https://medium.com/cryptium-cosmos/how-to-store-your-cosmos-atoms-on-your-ledger-and-delegate-with-the-command-line-929eb29705f_
 
-### Prepare your Linux host to work with ledger
+### Fix Connection Issues
+
+* Prepare your Linux host to work with ledger
```

Comment:
Thanks @lacabra ! :smile: 
Can you edit this line to be just "Linux"?

Reply:
@lacabra?